downloads.html

```diff
@@ -8,6 +8,28 @@
   - /downloads.php
   - /old_releases/
   - /other_distros/
+latest_version: 0.4.15
+latest_downloads:
+    windows:
+        os: Windows
+        title: portable, 64-bit
+        url: https://github.com/minetest/minetest/releases/download/0.4.15/minetest-0.4.15-win64.zip
+    mac:
+        os: OS X
+        title: Stable OS X Builds
+        url: https://forum.minetest.net/viewtopic.php?t=9190
+    ubuntu:
+        os: Ubuntu
+        title: search for .deb
+        url: http://packages.ubuntu.com/search?keywords=minetest
+    debian:
+        os: Debian
+        title: search for .deb
+        url: https://packages.debian.org/search?keywords=minetest
```

Comment:
It's worth noting that I literally copied the links from the Linux section, and this problem existed before. There's just less text now to warn them